src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -331,13 +331,6 @@ protected function frontendDevEnableDrupal(string $siteDir = 'default', array $o
             );
         }
 
-        $this->say("enablig twig.debug in development.services.yml.");
-        $devServices = Yaml::parseFile($this->devServicesPath);
-        $devServices['parameters']['twig.config'] = [
-            'debug' => true,
-            'auto_reload' => true,
-        ];
-        file_put_contents($this->devServicesPath, Yaml::dump($devServices));
```

Comment:
These configurations already exist in the `development.services.yml` file that I reviewed so it is no longer needed to add them here.